sw/device/lib/base/memory.c

```diff
@@ -54,7 +71,9 @@ void *memchr(const void *ptr, int value, size_t len) {
   }
   return NULL;
 }
+#endif
 
+#if !defined(HOST_BUILD)
 void *memrchr(const void *ptr, int value, size_t len) {
```

Comment:
So, uh, technically this is a glibc extension. I don't think it's a big deal, since musl provides it too... but I thought it was worth calling out.

Reply:
Oh great. I'll un-`#ifdef` this as we're not all on glibc platforms.